src/IR.cpp

```diff
@@ -653,6 +653,7 @@ const char *const intrinsic_op_names[] = {
     "mux",
     "popcount",
     "prefetch",
+    "profiling_marker",
```

Comment:
Why this name? It's confusing to figure out that it actually calls `halide_profiler_enable_instance` instead.

Reply:
I changed it to profiling_instance_enable_marker.

The suffix _marker is used to communicate from early lowering passes to later lowering passes where certain things are supposed to be injected. There's really only one place where this is used though - to mark the code just before the pipeline is going to run for real, so maybe I should use some other IR construct.